packages/atomic/cypress/integration/facet-standard.cypress.ts

```diff
@@ -0,0 +1,406 @@
+import {getAnalyticsAt, getApiRequestBodyAt} from '../utils/network';
+import {setUpPage, shouldRenderErrorComponent} from '../utils/setupComponent';
+import {
+  FacetSelectors,
+  createAliasShadow,
+  createAliasFacetUL,
+  createBreadcrumbShadowAlias,
+} from './facet-selectors';
+
+const facetProp = {
+  field: 'author',
+  label: 'Authors',
+};
+
+const compareAlphanumericalValues = (valueA: string, valueB: string) => {
+  if (valueA.toLowerCase() < valueB.toLowerCase()) {
+    return -1;
+  }
+  if (valueA.toLowerCase() > valueB.toLowerCase()) {
+    return 1;
+  }
+  return 0;
+};
+
+const sortCriteriaOption = {
+  automatic: 'automatic',
+  alphanumeric: 'alphanumeric',
+  occurrences: 'occurrences',
+  score: 'score',
+};
+
+function setupPager(field: string, label: string, option?: string) {
+  setUpPage(
+    `<atomic-breadcrumb-manager></atomic-breadcrumb-manager>
+     <atomic-facet field="${field}" label="${label}" ${option}></atomic-facet>`
+  );
+}
+
+function componentLoaded(label: string) {
+  cy.get(FacetSelectors.facetStandard).should('be.visible');
+  cy.get('@facetShadow').find('div:nth-child(1)').should('contain.text', label);
+  cy.checkA11y(FacetSelectors.facetStandard);
+}
+
+function validateNumberofValue(totalNumber: number) {
+  cy.wait(200);
+  cy.get('@facet_ul').find('li').its('length').should('eq', totalNumber);
+}
+
+async function facetValueShouldDisplayInBreadcrumb(
```

Comment:
You are using async keyword for the function, but no await.

Suggestion:
```ts
async function facetValueShouldDisplayInBreadcrumb(
  facetValueSelector: string,
  valueDisplayInBreadcrumbSelector: string
) {
  const text = await get(facetValueSelector)
                      .find('label span:nth-child(1)')
                      .invoke('text')

   cy.get('@breadcrumbFacet')
        .find(valueDisplayInBreadcrumbSelector)
        .should('be.visible')
        .contains(text);
}
```

Reply:
That's what I've tried whole yesterday,  but for some reason, cypress doesn't work. Any cypress calls (`cy.get` for example) after a promise will be skipped. 
Like in the code above, the call  ```cy.get('@breadcrumbFacet') .....``` will be skipped


Some ppl recommended with `Cypress.Promise `, but it doesn't work neither
@ThibodeauJF recommended with done() also doesn't solve the issue

Reply:
https://docs.cypress.io/faq/questions/using-cypress-faq.html#Can-I-use-the-new-ES7-async-await-syntax
Should we stop using async/await with cypress at all?

Reply:
Yes, their documentation seems pretty clear on that point.